media/open_zim.ksy

```diff
@@ -0,0 +1,356 @@
+meta:
+  id: open_zim
+  title: "(Open) Zeno IMproved"
+  application: 
+    - Kiwix
+    - zimlib
+  file-extension: zim
+  xref:
+    wikidata: Q784695
+  license: CC-BY-SA-3.0
+  encoding: utf-8
+  endian: le
+
+doc: |
+  A file format to store encyclopaedias of articles written in MediaWiki markup language.
+  Files for test: https://dumps.wikimedia.org/other/kiwix/zim/wikipedia/
+
+doc-ref:
+  - https://www.openzim.org/wiki/ZIM_file_format
+  - https://wiki.openzim.org/wiki/OpenZIM
+
+seq:
+  - id: signature
+    -orig-id: magicNumber
+    contents: [0x5A, 0x49, 0x4D, 0x04]
+    doc: Magic number to recognise the file format, must be 
+  - id: version
+    type: version
+    doc: Version of the ZIM file format
+  - id: uuid
+    type: uuid
+    doc: unique id of this zim file
+  - id: article_count
+    type: u4
+    doc: total number of articles
+  - id: cluster_count
+    type: u4
+    doc: total number of clusters
+  - id: url_pointer_list_ptr
+    -orig-id: urlPtrPos
+    type: u8
+    doc: position of the directory pointerlist ordered by URL
+  - id: title_pointer_list_ptr
+    -orig-id: titlePtrPos
+    type: u8
+    doc: position of the directory pointerlist ordered by Title
+  - id: cluster_pointer_list_ptr
+    -orig-id: clusterPtrPos
+    type: u8
+    doc: position of the cluster pointer list
+  - id: mime_list_ptr
+    -orig-id: mimeListPos
+    type: u8
+    doc: |
+      position of the MIME type list (also header size)
+      The MIME type list always follows directly after the header, so the mimeListPos also defines the end and size of the ZIM file header.
+      The MIME types in this list are zero terminated strings.
+  - id: main_page_idx
+    -orig-id: mainPage
+    type: u4
+    doc: main page or 0xffffffff if no main page
+  - id: layout_page_idx
+    -orig-id: layoutPage
+    type: u4
+    doc: layout page or 0xffffffffff if no layout page
```

Comment:
The value `0xff_ffff_ffff` stated in the comment would be the highest value of a 5-byte unsigned int. I suggest putting underscores in such long hex literals, it makes them more clear:

```suggestion
    doc: layout page or 0xffff_ffff if no layout page
```